mesheryctl/internal/cli/root/system/status.go

```diff
@@ -93,6 +93,10 @@ var statusCmd = &cobra.Command{
 			// List the pods in the MesheryNamespace
 			podList, err := utils.GetPods(client, utils.MesheryNamespace)
 
+			if len(podList.Items) == 0 {
+				log.Fatal("no pods up yet...try again in a while")
+			}
+
```

Comment:
If @piyushsingariya 's fixes work, then let's stick with that now unless something else is still breaking. If it is not, let's revert these changes.

Reply:
Yes done!

Reply:
Alright, reverted the change.